pkg/meta/utils/parser/alias_yaml.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"github.com/inspr/inspr/pkg/ierrors"
+	"github.com/inspr/inspr/pkg/meta"
+	yaml "gopkg.in/yaml.v2"
+)
+
+// YamlToAlias - deserializes the yaml to a meta.Alias struct
+func YamlToAlias(bytes []byte) (*meta.Alias, error) {
+	var alias *meta.Alias
+
+	if err := yaml.Unmarshal(bytes, &alias); err != nil {
+		return nil, ierrors.NewError().Message("Error parsing the file").Build()
```

Comment:
You can use err.Error( ) as the ierrors message

Reply:
Good idea